lms/static/sass/views/_decoupled-verification.scss

```diff
@@ -147,9 +147,14 @@
 
         .retake-photos {
             color: $blue;
-
-            &:hover {
-                cursor: pointer;
+            background:none!important;
+            border:0!important;
+            padding:0!important;
+            box-shadow: none;
+
+          &:hover {
```

Comment:
@cptvitamin Do you mean to add `text-decoration: underline;` on `focus`, because other all styles remains constant.